gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
-#Sat Jan 20 19:11:29 EST 2024
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-bin.zip
```

Comment:
Can we go to 8.10 @andhikayuana 
cc: @josikie

Reply:
I think yes, it's one of the newest, but is it compatible? @kcw-grunt

Reply:
checking on this

Reply:
sure, we can go using gradle 8.10.
for now we can keep as is for the android gradle plugin version which is 8.2.1. but the recommendation here https://developer.android.com/build/releases/gradle-plugin#updating-gradle, we should use the latest possible version of both Gradle and the plugin.

to bump up with the latest version of android gradle plugin (AGP), I think we need to create separate PR in the next iteration since we need to make sure with kotlin version compatible and also migrating to `.gradle.kts` for modern build script.